doc/ug/github_notes.md

```diff
@@ -301,7 +301,7 @@ import livereload
 
 ```
 
-In this case the master tree (between `<<<<<<< HEAD` and `=======`)
+In this case the `master` tree (between `<<<<<<< HEAD` and `=======`)
```

Comment:
This could be reworded as `the upstream tree`, I think. Or maybe something more invasive could improve the document. Maybe something like:

```markdown
In this case, the upstream repository's copy of `util/build_docs.py`
was modified to import `docgen.generate` rather than just `docgen`.
Because git couldn't automatically merge that change with the one we
have made, it gave up. The code between `<<<<<<< HEAD` and `=======`
represents the change in the upstream repository and the code between
`=======` and `>>>>>>>` represents the change in our copy.

These lines have to be edited to get the correct merged result and the
diff markers removed. There may be multiple points in the file where
fixes are needed. Once all conflicts have been addressed the file can
be `git add`ed and once all files addressed the rebase continued.
```